modules/hibernating-cluster-resume.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/hibernating-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hibernating-cluster-resume_{context}"]
+= Resuming a hibernated cluster
+
+You can resume a hibernated cluster within 90 days. You might have to approve certificate signing requests (CSRs) for the nodes to become ready.
+
+It can take around 45 minutes for the cluster to resume, depending on the size of your cluster.
+
+.Prerequisites
+
+* You hibernated your cluster less than 90 days ago.
++
+// TODO: What if they try to resume later than 90 days? What are the consequences?
```

Comment:
@vrutkovs Thoughts on whether we there are any consequences we should state on what happens if you leave a cluster hibernated for more than 90 days?

Reply:
It varies a lot on the hibernation period.
If the resume time is more than 2 years, users would need to construct a kubeconfig to get access to the cluster. If its between 180 days and 2 years it may take longer for operators to recover (and possibly more manual steps to recover). We found no issues when the cluster is hibernated between 90 and 180 days.

Lets mention 90 days here as that's the period we committed to support